backend/Makefile

```diff
@@ -48,14 +48,14 @@ db/new_migration_auto:
 
 db/connect:
 	# Assuming you've created a tunnel to the DB. Check the docs for information on how to do that
-	$(eval DB_PW = $(shell aws secretsmanager get-secret-value --secret-id corpora/backend/${DEPLOYMENT_STAGE}/database --region us-west-2 | jq -r '.SecretString | split(":") | .[-1] | split("@") | .[0]'))
+	$(eval DB_PW = $(shell aws secretsmanager get-secret-value --secret-id corpora/backend/${DEPLOYMENT_STAGE}/database --region us-west-2 | jq -r '.SecretString | match(":([^:]*)@").captures[0].string'))
```

Comment:
This change allows the use of `DEPLOYMENT_STAGE=rdev` and maintains compatibility with dev / staging / prod.

Reply:
rdev tunneling works too?